main.go

```diff
@@ -178,22 +179,27 @@ If no version is passed, the pulumi/{pkg,sdk} version will track the bridge`)
 
 	cmd.PersistentFlags().BoolVar(&context.InferVersion, "pulumi-infer-version", false,
 		`Use our GH issues to infer the target upgrade version.
-		If both '--target-version' and '--pulumi-infer-version' are passed,
-		we take '--target-version' to cap the inferred version. [Hidden behind PULUMI_DEV]`)
+If both '--target-version' and '--pulumi-infer-version' are passed,
+we take '--target-version' to cap the inferred version. [Hidden behind PULUMI_DEV]`)
```

Comment:
This help text in actuality hidden no matter what due to the `MarkHidden` below.

We could probably make this message less confusing - as written, it makes it sound like the _functionality_ is hidden behind PULUMI_DEV, which is not the case.

Reply:
Fixed in https://github.com/pulumi/upgrade-provider/pull/234/commits/cdf3287eac7078a344b0c9cc76338a3c36e8e74b